app/main.py

```diff
@@ -1,34 +1,130 @@
+import types
+from typing import List, Any
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    def __str__(self) -> str:
+        if self.is_alive:
+            return " \u25A1 "
+        else:
+            return " * "
+
+    def __repr__(self) -> str:
+        if self.is_alive:
+            return " \u25A1 "
+        else:
+            return " * "
+
+
+def generate_coordinates(
+    point1: tuple, point2: tuple, step: int = 1
+) -> List[tuple]:
```

Comment:
This looks too complicated, I'd suggest at least try to remove indents and simplify code this way - at max to change logic here